src/coreclr/jit/lower.cpp

```diff
@@ -2198,6 +2210,124 @@ GenTree* Lowering::LowerTailCallViaJitHelper(GenTreeCall* call, GenTree* callTar
     return result;
 }
 
+//------------------------------------------------------------------------
+// LowerCFGCall: Potentially lower a call to use control-flow guard. This
```

Comment:
copy pasta.  There's more in the internal comment, but it's probably not worth trying to rewrite it yet.